### PR TITLE
{lib}[GCCcore/8.2.0] libsamplerate v0.1.9

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2705.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/814cb2c9fa2aae8b923b61de9be73795 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2071.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d70c0e4955fc9c5ab9b9d9f7750bdefb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3113.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/55ec870b00f1424eea3b9ea2299a7284 for a full test report.

File: easybuild/easyconfigs/l/libsamplerate/libsamplerate-0.1.9-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = ['http://www.mega-nerd.com/libsamplerate/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['0a7eb168e2f21353fb6d84da152e4512126f7dc48ccb0be80578c565413444c1']
+
+builddependencies = [
+    ('binutils', '2.31.1'),
+    # FFTW is only required for tests
+    ('FFTW', '3.3.8', '', ('gompi', '2019a')),
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/l/libsamplerate/libsamplerate-0.1.9-GCCcore-8.2.0.eb]: is this supposed to work with an hierarchical MNS?

Comment (boegel) [reply easybuild/easyconfigs/l/libsamplerate/libsamplerate-0.1.9-GCCcore-8.2.0.eb]: Hmm, good question, I may have to also include `gompi` separately as dependency...

@Micket You have an HMNS setup, right? Can you submit a test report for this PR?

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.6.1810, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/c4ee3e35222b7229fc186fdecece287e for a full test report.

**Comment by boegel** [comment]
```
Module command 'module load FFTW/3.3.8' failed
```

Nice catch @migueldiascosta! :)

Included `gompi` as build dep in 56aab78893 which should fix the issue, please re-test @Micket?

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.6.1810, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/6cede119577c859a2c9f5d4bd9254671 for a full test report.

**Comment by boegel** [comment]
@migueldiascosta Should be good to go now...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3104.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/ef3f4d33023ce919b872715f23892114 for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @boegel!

**Pull request merged**